presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -53,6 +53,7 @@
     private boolean optimizeSingleDistinct = true;
     private boolean optimizerReorderWindows = true;
     private boolean pushTableWriteThroughUnion = true;
+    private boolean pageCompressionEnabled = true;
```

Comment:
I did that initially, but it was annoying having to set that flag everywhere, and I do want to set that flag for as many tests as possible. Is there an easy way to set config properties for all tests?